### DISCUSSION TITLE
paperless-NGX shows wrong version

### DISCUSSION THREAD

**Discussion by FirstS0ul** [tteck/Proxmox#2251] [Help Needed] [answered]
hi,

i updated my paperless-ngx with the helper scripts (thanks for that!).
today i installes the version 2.1.3. installation works and my paperless works. maybe?
so in the left corner the version says 2.0.1.
Why? What did i wrong. my browser cache is clean

thanks!

**Comment by FirstS0ul** [comment]
nobody? *push*

**Reply by AlexGeek77** [reply]
Last time that happened to me I needed to reboot the container.

**Reply by FirstS0ul** [reply]
rebootet the LXC no change.

**Reply by AlexGeek77** [reply]
What happens if you rerun the 'update' script?

**Comment by FirstS0ul** [comment, marked as answer]
Found it! Had to change my folder for the static files. an run 
`python3 manage.py collectstatic`
Works now!
